cmd/blockchaincmd/describe.go

```diff
@@ -275,8 +276,14 @@ func printAllocations(sc models.Sidecar, genesis core.Genesis) error {
 		t.Style().Options.SeparateRows = true
 		t.SetTitle("Initial Token Allocation")
 		t.AppendHeader(table.Row{"Description", "Address and Private Key", "Amount (10^18)", "Amount (wei)"})
-		for address := range genesis.Alloc {
-			amount := genesis.Alloc[address].Balance
+		for address, allocation := range genesis.Alloc {
+			if len(allocation.Code) > 0 {
+				continue
+			}
+			if address == common.HexToAddress(icmgenesis.MessengerDeployerAddress) {
+				continue
+			}
```

Comment:
Why do we skip the deployer address and any allocation with code here? Could be helpful to add a comment explaining

Reply:
probably we should add a new table for smart contracts.

Reply:
changing to a 0 balance check, and adding a section on deployed contracts